src/DynamoCoreWpf/UI/GuidedTour/dynamo_guides.json

```diff
@@ -151,5 +151,32 @@
 				}
 			}
 		]
+	},
+	{
+		"Name": "Packages",
+		"GuideSteps": [
+			{
+				"Sequence": 0,
+				"Name": "packages",
+				"Type": "tooltip",
+				"TooltipPointerDirection": "top_left",
+				"VerticalTooltipOffset": -50,
```

Comment:
Same here I think it should be renamed to PointerVerticalOffset or something like that.